.github/workflows/java-ci.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           path: artifacts
 
       - name: Publish Unit Test Results
-        uses: EnricoMi/publish-unit-test-result-action@8c0ddccd03f69215064876293657094714331b04
+        uses: EnricoMi/publish-unit-test-result-action@dc4b2f22e07d0995dffe65daf902598f695e8827
```

Comment:
Shouldn't we use v1.25?
See https://github.com/EnricoMi/publish-unit-test-result-action/releases

Reply:
That [is actually 1.25](https://github.com/EnricoMi/publish-unit-test-result-action/commit/dc4b2f22e07d0995dffe65daf902598f695e8827). Afaik @Fox32 once followed a suggestion to pin the exact commit sha as tags are not immutable.